RecoEgamma/EgammaTools/plugins/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
   <use   name="FWCore/Framework"/>
   <use   name="PhysicsTools/MVAComputer"/>
   <use   name="PhysicsTools/MVATrainer"/>
+  <use   name="PhysicsTools/PatAlgos"/>
```

Comment:
Then this entire endeavor cannot happen, as our code is current structured.
Those particular lines are from years ago anyway.

On Tue, Jun 30, 2015 at 11:52 PM, Slava Krutelyov <notifications@github.com>
wrote:

> In RecoEgamma/EgammaTools/plugins/BuildFile.xml
> <https://github.com/cms-sw/cmssw/pull/9659#discussion_r33628110>:
>
> > @@ -3,6 +3,7 @@
> >    <use   name="FWCore/Framework"/>
> >    <use   name="PhysicsTools/MVAComputer"/>
> >    <use   name="PhysicsTools/MVATrainer"/>
> > +  <use   name="PhysicsTools/PatAlgos"/>
>
> there should be no dependence of RECO code on PatAlgos
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9659/files#r33628110>.
>